### PR TITLE
CCFits

### PR DESCRIPTION
**Pull request by giacomov**
CCfits is a C++ wrapper for the `cfitsio` library from HEASARC:

https://heasarc.gsfc.nasa.gov/fitsio/ccfits/

### CODE REVIEW

File: recipes/ccfits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - test -d ${PREFIX}/include/CCfits
+    
+    - conda inspect linkages -p $PREFIX ccfits  # [not win]
+    - conda inspect objects -p $PREFIX ccfits  # [osx]
+
+   
+about:
+  summary: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. It is written in ANSI C++ and implemented using the C++ Standard Library with namespaces, exception handling, and member template functions.
+  home: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/
+  license: GPL2
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: Please include the `license_file` and the `license_family`.

Comment (giacomov) [reply recipes/ccfits/meta.yaml]: Thanks, I just did.

File: recipes/ccfits/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: `GPL-2.0`

**Comment by giacomov** [comment]
@CJ-Wright is everything fine now? Do I need to do anything else to move forward?

File: recipes/ccfits/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+     - cmake
+
+ run:
+     - cfitsio
+     - libgcc  # [linux]
+   
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libCCfits.so  # [linux]
+
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: Can you remove the extra newlines between the tests here?

File: recipes/ccfits/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - test -f ${PREFIX}/lib/libCCfits.dylib  # [osx]
+    
+    - test -d ${PREFIX}/include/CCfits
+    
+    - conda inspect linkages -p $PREFIX ccfits  # [not win]
+    - conda inspect objects -p $PREFIX ccfits  # [osx]
+
+   
+about:
+  summary: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. It is written in ANSI C++ and implemented using the C++ Standard Library with namespaces, exception handling, and member template functions.
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: Can you break up this line (so that it's under 80 characters?)

File: recipes/ccfits/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "2.5" %}
+{% set sha256 = "e542f39fa496417c401be7fbca715d566462cf4c56c83e3d43b6df3cb8d92105" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: CCfits-{{version}}.tar.gz
+  url: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/CCfits-{{version}}.tar.gz
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: Is this available on pypi or github? (It's fine if it isn't, it's just that those might be easier to maintain (especially if auto version bumping gets off the ground))

File: recipes/ccfits/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+    - conda inspect linkages -p $PREFIX ccfits  # [not win]
+    - conda inspect objects -p $PREFIX ccfits  # [osx]
+
+   
+about:
+  summary: CCfits is an object oriented interface to the cfitsio library. It is designed to make the capabilities of cfitsio available to programmers working in C++. It is written in ANSI C++ and implemented using the C++ Standard Library with namespaces, exception handling, and member template functions.
+  home: https://heasarc.gsfc.nasa.gov/fitsio/ccfits/
+  license: GPL-2.0
+  license_family: GPL
+  license_file: '{{ environ["SRC_DIR"] }}/License.txt'
```
Comment (CJ-Wright) [inline recipes/ccfits/meta.yaml]: Is there a documentation URL?

File: recipes/ccfits/meta.yaml (hunk 5 above)
Comment (giacomov) [reply recipes/ccfits/meta.yaml]: No, it is not, unfortunately...

**Comment by giacomov** [comment]
I messed up the rebase procedure. Closing this and restarting from scratch. Will be linking this in the new pull request. Sorry for the issue.

**Pull request closed without merging**